### DISCUSSION TITLE
Resubmit Notification on the Front End

### DISCUSSION THREAD

**Discussion by AbbeyDesign** [solspace/craft-freeform#397] [Questions & Answers] [answered]
Is it possible to resend notifications from the front end?

**Comment by kjmartens** [comment, marked as answer]
Hi @AbbeyDesign, unfortunately there aren't any built-in options to do this. You could potentially use the "Edit" feature, and it would resend email notifications that way, but I'm not sure how clunky of a solution that might be for you:

https://docs.solspace.com/craft/freeform/v3/overview/submission-editing.html

You could potentially use JS or CSS to hide the rest of the form except the Submit button...

**Reply by AbbeyDesign** [reply]
Thanks. Will take that route.
